lib/sortify.rb

```diff
@@ -1,37 +1,28 @@
+require 'whitelist_scope'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.